app/models/user.rb

```diff
@@ -1,12 +1,24 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
```

Comment:
- [ ] I see that you remove `:confirmable` for some reason. Kindly make sure to add `:confirmable` as the previous reviewer mentioned so the user can confirm their email after signup